src/Sentry.Maui/Internal/MauiEventsBinder.cs

```diff
@@ -96,15 +98,30 @@ private void OnApplicationOnDescendantAdded(object? _, ElementEventArgs e)
             case Page page:
                 HandlePageEvents(page);
                 break;
-            case Button button:
-                HandleButtonEvents(button);
+            default:
+                if (e.Element is VisualElement ve)
+                {
+                    foreach (var binder in _elementEventBinders)
+                    {
+                        binder.Bind(ve, OnBreadcrumbCreateCallback);
+                    }
+                }
                 break;
-
-                // TODO: Attach to specific events on more control types
         }
     }
 
-    private void OnApplicationOnDescendantRemoved(object? _, ElementEventArgs e)
+    internal void OnBreadcrumbCreateCallback(BreadcrumbEvent breadcrumb)
+    {
+        _hub.AddBreadcrumbForEvent(
+            _options,
+            breadcrumb.Sender,
+            breadcrumb.EventName,
```

Comment:
@jamescrosswell and what about `breadcrumb.ExtraData` I think you completely missed it.

Reply:
Good catch @albyrock87 - I've added https://github.com/getsentry/sentry-dotnet/issues/4252 to track this.